etc/spack/defaults/packages.yaml

```diff
@@ -29,6 +29,7 @@ packages:
       glu: [mesa-glu, openglu]
       glx: [mesa+glx, mesa18+glx, opengl]
       golang: [gcc]
```

Comment:
While we're here: Shouldn't this include `go`?

Reply:
Applied.
```diff
-      golang: [gcc]
+      golang: [go, gcc]
```